packages/app/src/app/pages/Dashboard/Content/routes/SyncedSandboxes/index.tsx

```diff
@@ -38,9 +38,7 @@ export const SyncedSandboxesPage = () => {
     }
 
     if (home) {
-      return viewMode === 'grid' && items.length
-        ? [{ type: 'new-repo' }, ...items]
```

Comment:
Does this mean that we can delete everything related to the `new-repo` type?

Reply:
right! I removed everything now